connstr/README.md

```diff
@@ -0,0 +1,145 @@
+# Connect
+
+Connect to Oracle Database using `sql.Open("godror", dataSourceName)` where
+`dataSourceName` contains options such as the user credentials, the database
+connection string, and other configuration settings.   
+It should be a [logfmt](https://brandur.org/logfmt)-encoded  parameter list.   
+For example:
+
+```
+db, err := sql.Open("godror", `user="scott" password="tiger" connectString="dbhost:1521/orclpdb1" 
+	poolSessionTimeout=42s configDir=/tmp/admin 
+	heterogeneousPool=false standaloneConnection=false`)
+```
+Other connection and driver options can also be used:
+```
+db, err := sql.Open("godror", `user="scott" password="tiger" 
+	connectString="dbhost:1521/orclpdb1?connect_timeout=2" 
+	poolSessionTimeout=42s configDir="/opt/oracle/configdir" 
+	heterogeneousPool=false standaloneConnection=false`)
+```
+
+All godror parameters ([see
+here](https://pkg.go.dev/github.com/godror/godror?tab=doc#pkg-overview)) should
+also be specified logfmt-ted.
+
+You can use `ConnectionParams` to properly build such a string:
+
+    var P godror.ConnectionParams
+	P.Username, P.Password = "scott", godror.NewPassword("tiger")
+	P.ConnectString = "dbhost:1521/orclpdb1?connect_timeout=2"
+	P.SessionTimeout = 42 * time.Second
+	P.SetSessionParamOnInit("NLS_NUMERIC_CHARACTERS", ",.")
+	P.SetSessionParamOnInit("NLS_LANGUAGE", "FRENCH")
+	fmt.Println(P.StringWithPassword())
+	db := sql.OpenDB(godror.NewConnector(P))
+
+Or if you really want to build it "by hand", user connstr.AppendLogfmt:
```

Comment:
Small typo: "user" -> "use".

Also, would it be better to rename `connstr` to `dsn`, if it is going to be passed to `Open()` as the dataSourceName parameter?

Reply:
Good catch! I'll rename it.